File: app/src/main/java/com/kalashnyk/denys/airqualitymonitoring/splash_operation/SplashActivity.java
package com.kalashnyk.denys.airqualitymonitoring.splash_operation;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;

import com.kalashnyk.denys.airqualitymonitoring.R;
import com.kalashnyk.denys.airqualitymonitoring.main_operation.MainActivity;

public class SplashActivity extends AppCompatActivity {

    private final int SPLASH_DISPLAY_LENGTH = 3000;

    @Override
    public void onCreate(Bundle icicle) {
        super.onCreate(icicle);
        setContentView(R.layout.activity_splash);

        new Handler().postDelayed(() -> {
            Intent mainIntent = new Intent(SplashActivity.this, MainActivity.class);
            SplashActivity.this.startActivity(mainIntent);
            SplashActivity.this.finish();
        }, SPLASH_DISPLAY_LENGTH);
    }
}

File: app/src/main/java/com/kalashnyk/denys/splash_operation/SplashActivity.kt
package com.kalashnyk.denys.splash_operation

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import com.example.myapplication.R
import com.kalashnyk.denys.main_operation.MainActivity


class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
    }

    override fun onResume() {
        val myIntent = Intent(this, MainActivity::class.java)
        super.onResume()
        Handler().postDelayed({
            startActivity(myIntent)
            finish()
        }, 3000)
    }
}